sky/cli.py

```diff
@@ -3063,16 +3063,35 @@ def _output():
                 return
 
         # Show detailed accelerator information
+
+        name, quantity = accelerator_str.split(':')[0], None
+        has_quantity = accelerator_str.count(':') > 0
+        if has_quantity:
+            quantity = int(accelerator_str.split(':')[1])
```

Comment:
We can improve readability and add validation for input here:
```suggestion
        accelerator_split = accelerator_str.split(':')
        if len(accelerator_split) > 2:
            raise click.UsageError(
                f'Invalid accelerator string {accelerator_str}. '
                'Expected format: <accelerator_name>[:<quantity>].')
        if len(accelerator_split) == 2:
            name = accelerator_split[0]
            # Check if quantity is valid
            try:
                quantity = int(accelerator_split[1])
            except ValueError:
                raise click.UsageError(
                    f'Invalid accelerator quantity {accelerator_split[1]}. '
                    'Expected an integer.')
        else:
            name, quantity = accelerator_str, None
```

Reply:
Added validation as suggested!